README.md

```diff
@@ -603,6 +603,9 @@ the cluster will not be formed unless you terminate the initial node.
 
 **Please note that this operation is disruptive, and there is a risk that things may not go as expected.**
 
+**Note:** Scaling down a cluster to a single node is not supported. To do this you need to manually scale down to `0`,
```

Comment:
I would suggest a change in the step order: delete cluster, manually scale down to `0` then to 1, afterwards, delete the remaining disks from the other two nodes.

Reply:
If you delete the cluster first, an update might land on a node that will later be removed. However, if you are going to do this, you might want to disable access.

Reply:
changed